<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ClientWindow</class>
    <widget class="QMainWindow" name="ClientWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>500</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Qt Chat Client</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QGroupBox" name="connectionGroup">
                        <property name="title">
                            <string>Connection</string>
                        </property>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Name:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="nameEdit">
                                    <property name="placeholderText">
                                        <string>Your name</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="text">
                                        <string>Server:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="serverEdit">
                                    <property name="text">
                                        <string>localhost</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_3">
                                    <property name="text">
                                        <string>Port:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="portEdit">
                                    <property name="text">
                                        <string>12345</string>
                                    </property>
                                    <property name="maximumWidth">
                                        <number>80</number>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="connectButton">
                                    <property name="text">
                                        <string>Connect</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="disconnectButton">
                                    <property name="text">
                                        <string>Disconnect</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QTextBrowser" name="messagesView">
                        <property name="html">
                            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot;
                                &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot;
                                content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
                                p, li { white-space: pre-wrap; }
                                &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans
                                Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
                                &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px;
                                margin-bottom:0px; margin-left:0px; margin-right:0px;
                                -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br
                                /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="messageGroup">
                        <property name="title">
                            <string>Message</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QTextEdit" name="messageEdit">
                                    <property name="placeholderText">
                                        <string>Type your message here...</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <item>
                                        <widget class="QPushButton" name="recordButton">
                                            <property name="text">
                                                <string>Hold to Record</string>
                                            </property>
                                            <property name="checkable">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="sendButton">
                                            <property name="text">
                                                <string>Send</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="callGroup">
                        <property name="title">
                            <string>Voice Call</string>
                        </property>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                                <widget class="QLabel" name="label_4">
                                    <property name="text">
                                        <string>Call to:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="callEdit">
                                    <property name="placeholderText">
                                        <string>User name</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="callButton">
                                    <property name="text">
                                        <string>Call</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="endCallButton">
                                    <property name="text">
                                        <string>End Call</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>